folioreader/src/main/java/com/folioreader/ui/activity/FolioActivity.kt

```diff
@@ -244,6 +245,7 @@ class FolioActivity : AppCompatActivity(), FolioActivityCallback, MediaControlle
 
         // Need to add when vector drawables support library is used.
         AppCompatDelegate.setCompatVectorFromResourcesEnabled(true)
+        DbAdapter.initialize(this)
```

Comment:
Using static classes that are initialized somewhere in the lifecycle is not a good practice at all. But it will require a lot refactoring to fix everything.